requirements.txt

```diff
@@ -6,7 +6,8 @@ colorama==0.3.7
 coverage==4.4.2
 flake8==3.5.0
 flake8-per-file-ignores==0.4
-freezegun==0.3.9
+freezegun==0.3.11
```

Comment:
any reason why freezgun version was updated and why you added httpretty on this PR?

Reply:
@zaro0508 that was my bad, I was building and testing under python 3.7.0, it's out of scope for this issue though so will remove it and offer it up elsewhere.  Thanks!